Z_Responses/GreatLitany-Romanian.ly

```diff
@@ -47,30 +46,32 @@ soprano = \relative g' {
   \once \override Score.RehearsalMark.self-alignment-X = #LEFT
   \mark "5 - Help us, save us.."
   c2( g4) a bes( g) f2 \bar "||"
-  % To Thee O Lord, uses anacrusis / pickup bar
-  \partial 4 c'4 g( a bes) g f2.\bar "||"
+  % To Thee O Lord
+  \partial 4 c'4 g( a bes) g f2. \bar "||"
+  % The last measure ended after 3 beats but we are in 4/4; this resets our measure position
+  \set Timing.measurePosition = #(ly:make-moment 0)
   % Amen
-  a4 a2. \bar "|."
+  a2 a2 \bar "|."
 }
 
 alto = \relative g' {
   \global
-  f4 f2.
+  f2 f2
   f2. f4 e2 c2
   f2. f4 f2 f2
   a4( f g) a f2 a
   f2( g4) a g2 f
   % Help us save us
-  f2. f4 f e c2
+  f2. f4 f( e) c2
   % To Thee O Lord
   f4 g( f g) e c2.
   % Amen
-  f4 f2.
+  f2 f2
 }
 
 tenor = \relative c' {
   \global
-  c4 c2.
+  c2 c2
   c2( bes4) a c2 a2
   c2( bes4) c d2 c2
   c2. c4 bes2 c
```

Comment:
I originally thought this was a mistake, as the copy we've been using has a D instead of a Bb, but it seems like some other arrangements do it this way. I think it's actually better how you have it here, because it's usually better to double the bass note (Bb) rather than the 3rd (D).

Reply:
Yea, I copied this one from the OCA site, and I think I like it better